api/test/wfl/integration/datarepo_test.clj

```diff
@@ -78,19 +78,17 @@
             (is (= 1 row_count))
             (is (= 0 bad_row_count))))))))
 
-(def ^:private testing-dataset "85efdfea-52fb-4698-bee6-eef76104a7f4")
+(def ^:private testing-dataset "4a5d30fe-1f99-42cd-998b-a979885dea00")
```

Comment:
While you're here, it might help us identify datasets if you include their name here (i.e. make this a map with a `:name` as well as an `:id`)

Reply:
The same for the snapshot too - I think the names are easier to remember (and not delete) than the uuids

Reply:
fair enough, a map sounds a great idea! working on it now!